src/extract-tools.ts

```diff
@@ -361,6 +361,14 @@ export function extractToolsFromApi(api: OpenAPIV3.Document, defaultInclude = tr
             const { inputSchema, parameters, requestBodyContentType } =
                 generateInputSchemaAndDetails(operation, pathItem.parameters);
 
+            if (typeof inputSchema === 'object' && inputSchema.properties) {
+                const propertiesEntries = Object.entries(inputSchema.properties).map(value => value[1]);
+                const missingDescriptions = propertiesEntries.filter((p: any) => p && typeof p === 'object' && !p.description);
+                if (missingDescriptions.length) {
+                    logs.push({ severity: "ERR", msg: "has parameter(s) with no `description` key" })
```

Comment:
I think we want to include operationId in the log to indicate which operation has parameters w/o description. Or is the operationId logged before this error msg?